# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../binutils/hotfix.patch
# Copyright (C) 2014 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- binutils-2.24/bfd/doc/bfd.texinfo.vanilla	2014-02-04 10:47:03.000000000 +0100
+++ binutils-2.24/bfd/doc/bfd.texinfo	2014-02-04 10:47:06.000000000 +0100
@@ -324,7 +324,7 @@
 @printindex cp
 
 @tex
-% I think something like @@@colophon should be in texinfo.  In the
+% I think something like @@@@colophon should be in texinfo.  In the
 % meantime:
 \long\def\colophon{\hbox to0pt{}\vfill
 \centerline{The body of this manual is set in}
